### PR TITLE
Updating prometheus doc for the new installation steps

### PR DESCRIPTION
**Pull request by rajdas98**
Refactoring doc for the new changes in the prometheus-service
Issue- keptn-contrib/prometheus-service#53

Signed-off-by: Raj Das mail.rajdas@gmail.com

### CODE REVIEW

File: content/docs/0.8.x/monitoring/prometheus/install/index.md [hunk 1]
```diff
@@ -13,33 +13,72 @@ In order to evaluate the quality gates and allow self-healing in production, we
...
+    value: 'alertmanager-templates'
+```
 
-* To install the *prometheus-service*, execute: 
+#### Execute the following steps to install prometheus-service
 
-    ```console
-    kubectl apply -f https://raw.githubusercontent.com/keptn-contrib/prometheus-service/$VERSION/deploy/service.yaml
-    ```
+* Download the Keptn's Prometheus service manifest
```
Comment (johannes-b) [inline content/docs/0.8.x/monitoring/prometheus/install/index.md:53]: Please change to: `Download the manifest of the prometheus-service:`

File: content/docs/0.8.x/monitoring/prometheus/install/index.md [hunk 2]
```diff
@@ -13,33 +13,72 @@ In order to evaluate the quality gates and allow self-healing in production, we
...
-    keptn configure monitoring prometheus --project=PROJECTNAME --service=SERVICENAME
-    ```
+* Replace the environment variable value according to the use case and apply the manifest
 
-## Verify Prometheus setup in your cluster
+```bash
+kubectl apply -f service.yaml
+```
 
+* Install Role and Rolebinding to permit Keptn's prometheus-service for performing operations in the Prometheus installed namespace.
```
Comment (johannes-b) [inline content/docs/0.8.x/monitoring/prometheus/install/index.md:65]: Please change to: `Install Role and RoleBinding to permit the prometheus-service for performing operations in the Prometheus installed namespace:`

File: content/docs/0.8.x/monitoring/prometheus/install/index.md [hunk 3]
```diff
@@ -13,33 +13,72 @@ In order to evaluate the quality gates and allow self-healing in production, we
 
 ## Setup Prometheus
 
-After creating a project and service, you can setup Prometheus monitoring and configure scrape jobs using the Keptn CLI. Therefore you need to deploy the *prometheus-service*. 
+After creating a project and service, you can set up Prometheus monitoring and configure scrape jobs using the Keptn CLI.
 
-* Specify the version of the prometheus-service you want to deploy. Please see the [compatibility matrix](https://github.com/keptn-contrib/prometheus-service#compatibility-matrix) of the prometheus-service to pick the version that works with your Keptn.  
+Keptn doesn't install or manage Prometheus and its components. Users need to install Prometheus and Prometheus Alert manager as a prerequisite.
```
Comment (johannes-b) [inline content/docs/0.8.x/monitoring/prometheus/install/index.md:18]: Please move this line to the `Prerequisites` above.

File: content/docs/0.8.x/monitoring/prometheus/install/index.md [hunk 4]
```diff
@@ -13,33 +13,72 @@ In order to evaluate the quality gates and allow self-healing in production, we
 
 ## Setup Prometheus
```
Comment (johannes-b) [inline content/docs/0.8.x/monitoring/prometheus/install/index.md:14]: Can you please change it to: `## Setup Prometheus Keptn integration`

File: content/docs/0.8.x/monitoring/prometheus/install/index.md (hunk 1 above)
Comment (rajdas98) [reply content/docs/0.8.x/monitoring/prometheus/install/index.md]: Done

File: content/docs/0.8.x/monitoring/prometheus/install/index.md (hunk 2 above)
Comment (rajdas98) [reply content/docs/0.8.x/monitoring/prometheus/install/index.md]: Done

File: content/docs/0.8.x/monitoring/prometheus/install/index.md (hunk 3 above)
Comment (rajdas98) [reply content/docs/0.8.x/monitoring/prometheus/install/index.md]: Done

File: content/docs/0.8.x/monitoring/prometheus/install/index.md (hunk 4 above)
Comment (rajdas98) [reply content/docs/0.8.x/monitoring/prometheus/install/index.md]: Done

File: content/docs/0.8.x/monitoring/prometheus/install/index.md [hunk 5]
```diff
@@ -10,36 +10,74 @@ In order to evaluate the quality gates and allow self-healing in production, we
 ## Prerequisites
 
 - Keptn project and at least one onboarded service must be available.
+- Keptn doesn't install or manage Prometheus and its components. Users need to install Prometheus and Prometheus Alert manager as a prerequisite.
 
-## Setup Prometheus
+## ## Setup Prometheus Keptn integration
```
Comment (johannes-b) [inline content/docs/0.8.x/monitoring/prometheus/install/index.md:15]: Please remove one of the `##` in this line.

File: content/docs/0.8.x/monitoring/prometheus/install/index.md [hunk 6]
```diff
@@ -10,36 +10,74 @@ In order to evaluate the quality gates and allow self-healing in production, we
...
+kubectl apply -f https://raw.githubusercontent.com/keptn-contrib/prometheus-service/release-0.5.0/deploy/role.yaml -n <PROMETHEUS_NS>
+```
+
+* Execute the following command to install Prometheus and set up the rules for the *Prometheus Alerting Manager*:
+
+```bash
+keptn configure monitoring prometheus --project=sockshop --service=carts
+```
+
+#### Optional: Verify Prometheus setup in your cluster
```
Comment (johannes-b) [inline content/docs/0.8.x/monitoring/prometheus/install/index.md:76]: Please change this line to: `## Optional: Verify Prometheus setup in your cluster`

File: content/docs/0.8.x/monitoring/prometheus/install/index.md [hunk 7]
```diff
@@ -10,36 +10,74 @@ In order to evaluate the quality gates and allow self-healing in production, we
...
+    value: 'default'
+    # Alert Manager template configmap name
+    - name: ALERT_MANAGER_TEMPLATE_CM
+    value: 'alertmanager-templates'
+```
 
-    ```console
-    VERSION=<VERSION>   # e.g.: VERSION=0.3.5
-    ```
+#### Execute the following steps to install prometheus-service
```
Comment (johannes-b) [inline content/docs/0.8.x/monitoring/prometheus/install/index.md:50]: Please change it to: `**Execute the following steps to install prometheus-service**`

**Comment by jetzlstorfer** [comment]
@johannes-b please have another look

**Review by johannes-b — APPROVED** [review]
lgtm

**Comment by johannes-b** [comment]
@rajdas98 many thanks for the changes. LGTM

**Pull request merged**